### PR TITLE
Created report controller to add, get, and get all reports from user

### PR DESCRIPTION
**Pull request by derrickhnguyen**

### CODE REVIEW

File: src/main/java/com/murgaloids/server/report/ReportController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@RestController
+@RequestMapping(path="/reports")
+public class ReportController {
+    @Autowired
+    private ReportRepository reportRepository;
+
+    @Autowired
+    private StudentRepository studentRepository;
+
+    @PostMapping("/add-report")
```
Comment (kuminin) [inline src/main/java/com/murgaloids/server/report/ReportController.java]: Would it be possible to change this to /add over /add-report?

File: src/main/java/com/murgaloids/server/report/ReportController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    @GetMapping("/get")
+    public @ResponseBody Report getReport(@NonNull @RequestParam Long reportId) {
+        if (reportRepository.existsById(reportId))
+            return reportRepository.findById(reportId);
+
+        return null;
+    }
+
+    @GetMapping("/get-all")
```
Comment (kuminin) [inline src/main/java/com/murgaloids/server/report/ReportController.java]: Same with line 27. Also this confuses me with line 50. Could you change this to "/get"?

File: src/main/java/com/murgaloids/server/report/ReportController.java (hunk 1 above)
Comment (derrickhnguyen) [reply src/main/java/com/murgaloids/server/report/ReportController.java]: Fixed

File: src/main/java/com/murgaloids/server/report/ReportController.java (hunk 2 above)
Comment (derrickhnguyen) [reply src/main/java/com/murgaloids/server/report/ReportController.java]: Fixed

**Pull request merged**